Apps/W1/HybridGP/app/src/codeunits/HybridGPWizard.codeunit.al

```diff
@@ -159,6 +160,9 @@ codeunit 4015 "Hybrid GP Wizard"
 
         if not HybridReplicationDetail.IsEmpty() then
             HybridReplicationDetail.DeleteAll();
+
+        if not GPMigrationErrorOverview.IsEmpty() then
+            GPMigrationErrorOverview.DeleteAll();
```

Comment:
We probably should not delete the errors on ResetAllCloudData. We can delete the records when the companies are deleted.